### PR TITLE
Fixes #33182 - Schedule recurring job for hostgroups

### PR DESCRIPTION
**Pull request by xprazak2**
Needs https://github.com/theforeman/foreman_remote_execution/pull/661

### CODE REVIEW

**Comment by xprazak2** [comment]
Seems like everything is working with the search on purpose, so the targeting search will probably not be needed.

File: app/views/foreman_ansible/ansible_roles/_hostgroup_ansible_roles_button.erb [hunk 1]
```diff
@@ -5,11 +5,14 @@
     display_link_if_authorized(_('Run all Ansible roles'), hash_for_play_roles_hostgroup_path(id: hostgroup), :'data-no-turbolink' => true, title: _('Run all Ansible roles on hosts belonging to this host group'))
   end
 
+  assign_jobs = link_to(_("Configure Ansible Job"), "/ansible/hostgroups/#{hostgroup.id}", { class: 'la' })
```
Comment (ezr-ondrej) [inline app/views/foreman_ansible/ansible_roles/_hostgroup_ansible_roles_button.erb:8]: Shouldn't we check for permissions here?

Comment (xprazak2) [reply app/views/foreman_ansible/ansible_roles/_hostgroup_ansible_roles_button.erb:8]: We probably should, changed.

**Review by ezr-ondrej — APPROVED** [review]
It's a good work around, this will have to be refactored once we have actual Hostgroup details page :)
Thanks @xprazak2 ! :+1:

**Pull request merged**